packages/backend/src/storage/storage.ts

```diff
@@ -294,17 +296,29 @@ export class Storage {
     if (repo && !repo.eventsAttached) {
       log('Subscribing to channel ', channelData.address)
 
-      db.events.on('write', (_address, entry) => {
+      db.events.on('write', async (_address, entry) => {
+        const message = entry.payload.value
+        const crypto = getCrypto()
+        const signature = stringToArrayBuffer(message.signature)
```

Comment:
Looks like this new code is the same as in the event below. It should be at least put into a separate function.